src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -81,6 +82,24 @@ public Task GetRevisionsFor()
             }
         }
 
+        [RavenAction("/databases/*/revisions/revert", "GET", AuthorizationStatus.ValidUser)]
+        public async Task Revert()
+        {
+            var operationId = GetLongQueryString("operationId");
+            var time = GetDateTimeQueryString("point-in-time");
+            var window = GetTimeSpanQueryString("window", required: false) ?? TimeSpan.FromHours(48);
```

Comment:
If we consider this a 'time machine' feature, then the only value that would make that happen (keep the consistency) would be 'infinite' ?

Reply:
You mean the default value? 
The revisions aren't sorted by date so theoretically we need to iterate over everything. 
So we are using the `window` for performance sake.

Reply:
Yes I understand that, but using any other value than infinite do not guarantee you consistency? That is my concern.